src/onResize/attachMarks.js

```diff
@@ -0,0 +1,6 @@
+export default function attachMarks() {
+    this.marks.forEach(mark => {
```

Comment:
Not sure what this bit is doing really ...

Reply:
Is it actually called anywhere else?

Reply:
yeah, this just makes accessing `chart.points` and `chart.lines` more accessible down the line, instead of having to `chart.marks.find(mark => mark.type == 'circle|line')` every time they need accessing